src/components/KeyboardShortcutRow.jsx

```diff
@@ -67,12 +67,12 @@ export default withStyles(({ reactDates: { color } }) => ({
   KeyboardShortcutRow_keyContainer: {
     display: 'inline-block',
     whiteSpace: 'nowrap',
-    textAlign: 'right',
-    marginRight: 6,
+    textAlign: 'right', // is not handled by isRTL
```

Comment:
Does that mean these should have a todo?

Reply:
I think once we got to the keyboard shortcuts, we realized that this panel has just never been considered for RTL. So maybe? I think that is an open question.